### DISCUSSION TITLE
Panda3DS post-Halloween edition

### DISCUSSION THREAD

**Discussion by wheremyfoodat** [wheremyfoodat/Panda3DS#375] [Announcements]
Wake up honey, new Panda3DS release just dropped and the changelog is so big I apparently can't autogenerate it.

More games working, first release including Android builds, new desktop UI (Labeled Qt below, as opposed to the original SDL frontend which is just a window), and more

<hr /><em>This discussion was created from the release <a href='https://github.com/wheremyfoodat/Panda3DS/releases/tag/v0.8'>Panda3DS post-Halloween edition</a>.</em>

**Comment by fleroviux** [comment]
:rpog:

**Comment by skylersaleh** [comment]
:rpog:

**Comment by ghost** [comment]
I think it's a little too late for even post-halloween lol

**Comment by HJMBgit84** [comment]
I tested the android build but it is not working yet, I just see the controls over a black screen

**Comment by notprogramminggames** [comment]
Should've named it post Christmas.

**Comment by J66788777** [comment]
What's the difference between Android x64 and Arm64?

**Reply by wheremyfoodat** [reply]
Different builds for different CPU architectures. If you're unsure about what you have, you probably have an arm64 device.

**Reply by GabrielBRDeveloper** [reply]
arm64 is for default android devices (99.9% of devices).
x64 is for android devices with intel/amd processors (emulators, vms, pc, a some rare devices).

**Reply by J66788777** [reply]
What's the most recent Devices with x64 Chip? Tried looking it up but all of them are outdated. Those can run Panda3DS well?!
